x-pack/plugins/secops/public/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IModule, IScope } from 'angular';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { UIRoutes as KibanaUIRoutes } from 'ui/routes';
+
+import {
+  AppBufferedKibanaServiceCall,
+  AppFrameworkAdapter,
+  AppKibanaAdapterServiceRefs,
+  AppKibanaUIConfig,
+  AppTimezoneProvider,
+  AppUiKibanaAdapterScope,
+} from '../../lib';
+
+const ROOT_ELEMENT_ID = 'react-secops-root';
+const BREADCRUMBS_ELEMENT_ID = 'react-secops-breadcrumbs';
+
+export class AppKibanaFrameworkAdapter implements AppFrameworkAdapter {
+  public appState: object;
+  public dateFormat?: string;
+  public kbnVersion?: string;
+  public scaledDateFormat?: string;
+  public timezone?: string;
+
+  private adapterService: KibanaAdapterServiceProvider;
+  private timezoneProvider: AppTimezoneProvider;
+  private rootComponent: React.ReactElement<any> | null = null;
+  private breadcrumbsComponent: React.ReactElement<any> | null = null;
+
+  constructor(uiModule: IModule, uiRoutes: KibanaUIRoutes, timezoneProvider: AppTimezoneProvider) {
+    this.adapterService = new KibanaAdapterServiceProvider();
+    this.timezoneProvider = timezoneProvider;
+    this.appState = {};
```

Comment:
Unless I'm missing something, `this.appState = {}` here and in the Infra project isn't used. I deleted it locally and nothing complained.

Reply:
That's funny because I deleted too and put it back because I was scared that I missed something too
I will delete it